matchtree.go

```diff
@@ -1115,7 +1115,7 @@ func (d *indexData) newMatchTree(q query.Q, opt matchTreeOpt) (matchTree, error)
 
 		return &symbolRegexpMatchTree{
 			regexp:    regexp,
-			all:       regexp.String() == "(?i)(?-s:.)*",
+			all:       regexp.String() == "(?i)(?-s:.*)",
```

Comment:
we should probably not be looking at string output here to decide this. This is super fragile.